### PR TITLE
suppressing client side polling

### PR DESCRIPTION
**Pull request by wheresrhys**
The api's getting a little bit messy now. Would benefit from a major release but for now just want to stop these https://app.getsentry.com/nextftcom/ft-next-frontend/group/63553639/ @matthew-andrews

### CODE REVIEW

**Comment by matthew-andrews** [comment]
can we remove the polling logic entirely from the client side code?

**Comment by matthew-andrews** [comment]
that way won't be breaking change [kinda]

**Comment by wheresrhys** [comment]
need to add client side tech debt alerts. e.g.

```
alerter = function () {
  if (this.get('clientSideErrorReporting') {
    raven.captureMessage ...
}
}
```

File: main.js [hunk 1]
```diff
@@ -1,39 +1,13 @@
...
 
-// Cookie looks like this: next-flags=ads:on,tearsheets:off
-var regex = new RegExp('(?:^|;)\\s?next-flags=(.*?)(?:;|$)', 'i');
-var match = document.cookie && document.cookie.match(regex);
-var rawOverrides = '';
-var overrides = {};
-if (match) {
-	rawOverrides = match[1];
+if (!window.nextFeatureFlags) {
+	throw 'Cannot initialise client sside feature flags without window.nextFeatureFlags object';
```
Comment (matthew-andrews) [inline main.js]: `client sside` => `client side`

**Comment by matthew-andrews** [comment]
weird still seems to be running obt

**Comment by matthew-andrews** [comment]
apart from failing tests :+1:

**Pull request merged**